dirac.cfg

```
@@ -1,3 +1,115 @@
+    Servers = https://server:8443/Configuration/Server
+    #Servers +=
+
+    # The URL of the Master Configuration Server. This server is used for updating the Configuration Service
+    MasterServer = https://server:8443/Configuration/Server
+
+    # Enables automatic merging of the modifications done in parallel by several clients. Takes a boolean value. By default false.
+    #EnableAutoMerge = false
+
+    # This subsection is used to configure the Configuration Servers attributes. It should not edited by hand since it is
+    # upated by the Master Configuration Server to reflect the current situation of the system. Takes a boolean value. By default true.
+    #AutoPublish = true
+
+    # Name of Configuration file
+    Name = Dirac-Prod
+
+    # Set propagation time, by default 300 seconds.
+    #PropagationTime = 300
+
+    # How many time the secondary servers are going to refresh configuration from master.
+    # Expressed as Integer and seconds as unit. By default 300.
+    #RefreshTime = 300
+
+    # Set slaves grace time in a seconds. By default 600.
+    #SlavesGraceTime = 600
+
+    # CS configuration version used by DIRAC services as indicator when they need to reload the
+    # configuration. Expressed using date format. By default 0.
+    #Version = 2011-02-22 15:17:41.811223
+  }
+
+  # This subsection defines several options related to the DIRAC security framework.
+  # WARNING: This section should only appear in the local dirac.cfg file of each installation, never in the central configuration.
+  Security
+  {
+    # Flag to use server certificates and not user proxies. This is typically true for the server installations. By default false.
+    #UseServerCertificate = true
+
+    # Flag to skip the server identity by the client. The flag is usually defined in the client installations. By default false.
+    #SkipCAChecks = false
+
+    # Path where the host certificate is located on the server. If not specified, DIRAC will try to find it.
+    #CertFile = /opt/dirac/etc/grid-security/hostcert.pem
+
+    # Path where the host key is located on the server. If not specified, DIRAC will try to find it.
+    #KeyFile = /opt/dirac/etc/grid-security/hostkey.pem
+
+    # Flag to use access tokens and not user proxies. This is typically false.
+    #UseTokens = true
+
+    # Section that describes OAuth 2.0 authorization settings and metadata, it is required to interact with the DIRAC Authorization Server. Please see https://datatracker.ietf.org/doc/html/rfc8414 for more details.
+    Authorization
+    {
+      # The authorization server's issuer identifier, which is a URL that uses the "https" scheme and has no query or fragment components. Please see https://datatracker.ietf.org/doc/html/rfc8414#section-2.
+      # This option must be defined in the client installations and on the DIRAC Authorization Server host, based on it other Authorization Server metadata names can be defined automatically.
```

```suggestion
      # This option must be defined in the client installations and on the DIRAC Authorization Server host.
```
Do I understand correctly, that the Issue URL is used inside the DIRAC (or OAuth) code to extract other information? Then I don't think you need to mention this here.

---

First running the `DIRAC Authorization Server` (AS), it reads the `DIRAC/Security/Authorization` section and subsections to generate its metadata according to the **RFC**. Then when the `DIRAC client` is installed, this metadata is read from the **_https://<DIRAC AS issuer>/.well-known/oauth-authorization-server_** request (https://datatracker.ietf.org/doc/html/rfc8414#section-3.1).

---

> based on it other Authorization Server metadata names can be defined automatically

means that when DIRAC AS generates its metadata, DIRAC AS issuer enough to do so

---

So this phrase can be dropped?

---

This is mentioned in the comments, code and documentation, so I think it can be removed here.